website/server/controllers/api-v3/groups.js

```diff
@@ -166,6 +166,12 @@ api.getGroup = {
     }
 
     group = Group.toJSONCleanChat(group, user);
+
+    if (group.purchased.plan.customerId) group.purchased.active = true;
```

Comment:
Yeah but you defined it into the model https://github.com/HabitRPG/habitica/pull/8153/files#diff-b6f239870392ed0ec2a2b73cbebaa699R97. If it's a pseudo field it should not be added to the model but to the JSON representation. You can use the toJSONTransform function